java/change-notes/2021-04-26-xpath-injection-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The query "XPath injection" (`java/xml/xpath-injection`) has been promoted from experimental to the main query pack. Its results will now appear by default. This query was originally [submitted as an experimental query by @SpaceWhite](https://github.com/github/codeql/pull/2800)
```

Comment:
One question here - has the query been added to the query suite files for LGTM and Code scanning? I don't see any changes to `.qls` files in this PR.

Reply:
Wasn't there a PR check for that at some point?

Reply:
I've lost track of how query suite files are managed these days.

Reply:
That rings a bell but I've also forgotten. Maybe it hasn't been reimplemented after we moved to Actions?

Reply:
I believe that new queries are added to code scanning suites automatically (via https://github.com/github/codeql/blob/main/misc/suite-helpers/code-scanning-selectors.yml and similar "helpers")

Reply:
Thanks Shati 😄 That's great to know.